.github/workflows/pull_request.workflow.yml

```diff
@@ -85,9 +85,6 @@ jobs:
      name: Documentation Snippet Tests
      needs: [unit-tests]
      runs-on: ubuntu-20.04
-     strategy:
-      matrix:
-        node-version: ["16", "18"]
```

Comment:
This is useless since the version used to test the snippets is the version set in the docker compose file above